py3status/formatter.py

```diff
@@ -623,6 +623,8 @@ def render(self, get_params, module, _if=None):
                 or getattr(module, threshold_color_name, None)
                 or getattr(module.py3, color_name.upper(), None)
             )
+            if color == "invisible":
```

Comment:
I only think it should say `hidden` instead of invisible, otherwise love this PR!

Reply:
(imho) I think `invisible` makes more sense when used with `color`. `hidden` is more suitable for `\?block=hidden` or `\?if=block=hidden`.

* Specifying `format = '\?color=hidden Hello, worlds!` seems a bit strange to me.
* Specifying `format = '\?color=invisible Hello, worlds!` seems better to me. 

Also, there are bunch of search results about human, birds, animals seeing invisible colors which kinda make sense to use `color=invisible` too.

Reply:
(I'm fine with both, as long as I can get this beautiful PR in master)